bedrock/firefox/templates/firefox/channel.html

```diff
@@ -24,7 +24,7 @@
 
 {% block body_id %}channel{% endblock %}
 
-{% block extrahead %}
+{% block page_css %}
```

Comment:
Should this page remain using `extrahead`, since it's a download page and older browser support is more critical?